test_onnx.py

```
@@ -0,0 +1,186 @@
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)